src/components/filesystem/index.js

```diff
@@ -758,81 +815,50 @@ module.exports = {
                 done:0,
                 max:resultingSize
             };
-            this.progressMonitors.push(progress);
-            var reader = new browserio.JSFileReader(file);
-            var java_reader = new peergos.shared.user.fs.BrowserFileReader(reader);
             var that = this;
-            var context = this.getContext();
 
             var updateProgressBar = function(len){
                 progress.done += len.value_0;
-                that.progressMonitors.sort(function(a, b) {
-                  return Math.floor(b.done / b.max) - Math.floor(a.done / a.max);
-                });
+                //that.progressMonitors.sort(function(a, b) {
```

Comment:
Why is this commented? I'd like to see the most progressed one on top, as otherwise it can be off screen

Reply:
I'm merging we can address this in another PR